stack.Rmd

```diff
@@ -277,7 +277,7 @@ We are using it in this class to reduce the burden of local installations, but i
   3. Launch a system shell and type in
     
     ```
-    docker run -e PASSWORD=yourpassword --rm --publish=8787:8787 rocker/verse
+    docker run --edit PASSWORD=yourpassword --rm --publish=8787:8787 rocker/verse
```

Comment:
If I run this in the terminal it fails with:

```
docker run --edit PASSWORD=yourpassword --rm --publish=8787:8787 rocker/verse
unknown flag: --edit
See 'docker run --help'.
```

I am guessing that this is because `--edit` is actually *not* the long form for `-e`. 

On the [`docker run` man page](https://www.mankier.com/1/docker-run#-e) it says that `--env` is the long form for `-e`, so it should be that.

Once you have added that it would also be good to just test run the command in your terminal (if possible on windows).